camera/cameraDisplayModule.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import robomodules as rm
+from messages import message_buffers, MsgType
+
+ADDRESS = os.environ.get("BIND_ADDRESS","localhost")
+PORT = os.environ.get("BIND_PORT", 11297)
+
+FREQUENCY = 10
+
+class CameraDisplayModule(rm.ProtoModule):
+    def __init__(self, addr, port):
+        self.subscriptions = [MsgType.CAMERA_MSG]
+        super().__init__(addr, port, message_buffers, MsgType, FREQUENCY, self.subscriptions)
+        self.value = -1
+        self.sub_ticks = 0
+        self.subbed = True
```

Comment:
`value`, `sub_ticks` and `subbed` are leftovers from the mock module. Remove them